code/modules/shuttle/shuttle.dm

```diff
@@ -618,7 +618,7 @@
 			if(!AM.anchored)
 				step(AM, dir)
 			else
-				if(AM.simulated) // Don't qdel lighting overlays, they are static
+				if(AM.simulated && !istype(AM, /obj/singularity)) // Don't qdel lighting overlays, they are static
```

Comment:
I feel like there might be a more OOP solution to this, but I have not slept in over 36 hours.

Reply:
I could perhaps make a `shuttle_crush_act` and on `atom` and override it depending on the type.

Reply:
Go sleep 😳